infra/service.tf

```diff
@@ -31,10 +31,36 @@ resource "google_cloud_run_v2_service" "server" {
           }
         }
       }
+      env {
+        name  = "PYTHONPATH"
+        value = ""
+      }
+      env {
+        name  = "DJANGO_SETTINGS_MODULE"
+        value = "avocano_api.settings"
+      }
+      env {
+        name  = "OTEL_METRICS_EXPORTER"
+        value = "none"
+      }
+      env {
+        name  = "OTEL_TRACES_EXPORTER"
+        value = "gcp_trace"
+      }
```

Comment:
These env variables are fairly cryptic. 

Is there some documentation about these that we could link to?

Reply:
i've added a link for the OTEL variables, since those seem most relevant. Did you want links for the others as well, or is this sufficient?